MeetupPlatform.Api/MeetupPlatform.Api.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="BCrypt.Net-Next" Version="4.0.3" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.3" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="6.0.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.4" />
```

Comment:
You don't need `NewtonsoftJson` here, please stick to default solution that comes out of the box (`System.Text.Json`).